File: webflux-dialogue/src/main/java/com/study/webflux/rag/application/dialogue/dto/SttTranscriptionResponse.java
package com.study.webflux.rag.application.dialogue.dto;

import io.swagger.v3.oas.annotations.media.Schema;

@Schema(description = "STT 변환 결과")
public record SttTranscriptionResponse(
	@Schema(description = "음성에서 추출한 텍스트", example = "오늘 회의 일정 알려줘")
	String transcription
) {
}

File: app-miyou/src/main/kotlin/com/miyou/app/infrastructure/inbound/web/dialogue/dto/SttTranscriptionResponse.kt
package com.miyou.app.infrastructure.inbound.web.dialogue.dto

import io.swagger.v3.oas.annotations.media.Schema

@Schema(description = "STT 변환 결과")
data class SttTranscriptionResponse(
    @field:Schema(description = "요청한 오디오에서 추출된 문자", example = "안녕하세요, 어떻게 도와드릴까요?")
    val transcription: String,
)
